cmd/network/verification.go

```diff
@@ -790,3 +800,14 @@ func printVersion() {
 	}
 	log.Println(fmt.Sprintf("Using osd-network-verifier version %v", version))
 }
+
+// Generate LimitedSupportTemplate
+func generateLimitedSupportTemplate(out *output.Output) lsupport.Post {
+	failures := out.GetEgressURLFailures()
+	if len(failures) > 0 {
+		return lsupport.Post{
+			Template: LimitedSupportTemplate,
+		}
+	}
+	return lsupport.Post{}
+}
```

Comment:
Do we even need this function? I'm not even sure we would want to return `lsupport.Post{}`, ever. 
We also already check for egress count before calling this. 

Can we just do the following in the above logic, instead of having `generateLimitedSupportTemplate`? 
```
pCmd := lsupport.Post{Template: LimitedSupportTemplate}
if err := pCmd.Run(e.ClusterId); err != nil {
   fmt.Printf("failed to post limited support reason: %v", err)
}
```

Reply:
Applied the above changes.